src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.tsx

```diff
@@ -95,29 +98,72 @@ const AdvancedSearchResultDetailsList: React.FC<IAdvancedSearchResultDetailsList
         }
     });
 
+    const getNoDataHeaderText = useCallback(() => {
+        if (additionalColumns.length) {
+            return 'No twins found';
+        } else {
+            return 'Add search criteria to get started';
+        }
+    }, [additionalColumns.length]);
+
+    const getNoDataDescriptionText = useCallback(() => {
+        if (additionalColumns.length) {
+            return 'Refine your search and try again';
+        } else {
+            return undefined;
+        }
+    }, [additionalColumns.length]);
+
+    const getNoDataImage = useCallback(() => {
+        if (additionalColumns.length) {
+            return {
+                height: 100,
+                src: NoResult
+            };
+        } else {
+            return undefined;
+        }
+    }, [additionalColumns.length]);
+
     return (
         <section className={classNames.root}>
             <h3 className={classNames.listHeader}>
                 {t('advancedSearch.results', { twinCount })}
             </h3>
-            <DetailsList
-                items={twins}
-                columns={
-                    additionalColumns.length > 0
-                        ? columns.concat(additionalColumns)
-                        : columns
-                }
-                layoutMode={DetailsListLayoutMode.justified}
-                selectionPreservedOnEmptyClick={false}
-                ariaLabelForSelectionColumn={t(
-                    'advancedSearch.toggleSelection'
-                )}
-                checkButtonAriaLabel={t('advancedSearch.selectRow')}
-                onRenderItemColumn={renderItemColumn}
-                selectionMode={SelectionMode.single}
-                selection={selection}
-                styles={classNames.subComponentStyles.detailsList}
-            />
+            {twinCount === 1000 && (
```

Comment:
please pull the 1000 out to a constant that you also use in generating the query so they stay in sync.